vm/src/exceptions.rs

```diff
@@ -862,13 +862,30 @@ fn key_error_str(exc: PyBaseExceptionRef, vm: &VirtualMachine) -> PyStrRef {
 
 fn os_error_str(exc: PyBaseExceptionRef, vm: &VirtualMachine) -> PyResult<PyStrRef> {
     let args = exc.args();
+    let obj = exc.as_object().to_owned();
```

Comment:
```suggestion
    let obj = exc.into();
```
otherwise if obj type is not inferrable,

```suggestion
    let obj: PyObjectRef = exc.into();
```

Reply:
This doesn't work since it will take the ownership of `exc`.

```
error[E0382]: borrow of moved value: `exc`
    |
869 |         let errno = exc.get_arg(0).unwrap().str(vm)?;
    |                     ^^^^^^^^^^^^^^ value borrowed here after move
```